dev/dependencies.sh

```diff
@@ -186,6 +187,7 @@ case "$MODULE" in
     SBT_PROJECT="celeborn-client-flink-1_17"
     ;;
   *)
+    MODULE="server"
```

Comment:
Because the `MODULE` parameter is not required, `DEP` and `DEP_PR` may refer to the empty string `MODULE`.